dist/build_deploy.sh

```diff
@@ -26,10 +26,6 @@ function cleanup() {
 }
 trap cleanup EXIT
 
-docker_cargo cargo build --release
-mkdir $RELEASE_OUTPUT_DIR
-cp ${RELEASE_DIR}/{graph-builder,policy-engine} $DOCKERFILE_DEPLOY  $RELEASE_OUTPUT_DIR/
```

Comment:
Does this mean we want to drop the other `DOCKERFILE_DEPLOY` references from this script?  It's not clear to me how it fits into the app-sre build pipeline.  Does it end up just being `dist/build.sh`?  Which is currently a wrapper of some sort around `build_deploy.sh`?  I'm fuzzy on how all of this is supposed to fit together.

Reply:
>Which is currently a wrapper of some sort around build_deploy.sh? I'm fuzzy on how all of this is supposed to fit together.

Previously we had a Jenkins job which ran `dist/build.sh`, it has been unsetting quay env vars so that the built image would not be pushed. The job has been removed so now `build_deploy.sh` is the only script used and it has some dead code.

I'd rather rework the internal CI procedure in a different PR really

Reply:
> I'd rather rework the internal CI procedure in a different PR really

Please add a comment to the script which code is stale.